win32/src/threadpool_win32.c

```diff
@@ -182,14 +153,32 @@ THANDLE(THREADPOOL) threadpool_create(EXECUTION_ENGINE_HANDLE execution_engine)
             }
             else
             {
-                /* Codes_SRS_THREADPOOL_WIN32_42_027: [ threadpool_create shall increment the reference count on the execution_engine. ]*/
-                execution_engine_inc_ref(execution_engine);
-                result->execution_engine = execution_engine;
+                /* Codes_SRS_THREADPOOL_WIN32_01_026: [ threadpool_open shall initialize a thread pool environment by calling InitializeThreadpoolEnvironment. ]*/

```

Comment:
>threadpool_open [](http://example.com/codeflow?start=56&length=15)

Meh, threadpool_open does not exist anymore.

Reply:
Changed.